cairo/legacy/utils/bytes.cairo

```diff
@@ -184,6 +184,34 @@ func bytes_to_felt(len: felt, ptr: felt*) -> felt {
     return current;
 }
 
+// @notice Loads a sequence of bytes into a single felt in little-endian.
+// @param len: number of bytes.
+// @param ptr: pointer to bytes array.
+// @return: packed felt.
+func bytes_to_felt_le(len: felt, ptr: felt*) -> felt {
```

Comment:
Assumption should be added
`tempvar current = current * 256 + [ptr + len]; could overflow at some point
`